FreeRTOS/FreeRTOS-Kernel
# Reintroduce Espressif's IDF v4.2 changes to ESP32 port

Reintroduce Espressif's IDF v4.2 changes to ESP32 port #92 

Description
-----------
With v4.2 of Espressif's IDF we can go ahead and re-merge their updated port.

Test Steps
-----------
Compile and run the new port on ESP32 and ESP32s2 with IDF v4.2.

Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
